src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -35,6 +36,7 @@ public class AddCommand extends Command {
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_ROLE + "friends "
             + PREFIX_ROLE + "owesMoney"
+            + PREFIX_METADATA + "I have food allegeries - Egg and Peanuts"
```

Comment:
Formatting error: `"I have food allegeries - Egg and Peanuts"` should be `"I have food allegeries - Egg and Peanuts "`